courses/machine_learning/intro_to_tensorflow/3_keras_sequential_api.ipynb

```diff
@@ -0,0 +1,482 @@
+{
```

Comment:
<p>To be able to inspect the model in Tensorboard, I'd add:</p><p> </p><p>at the beginning: </p><p>from tensorflow.keras.callbacks import TensorBoard</p><p> </p><p>then here I'd pass "[TensorBoard(LOGDIR)]" to the callbacks argument of the fit method. </p><p> </p><p>Then we can launch tensorboard on the LOGDIR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>